solo/fido2/__init__.py

```diff
@@ -1,9 +1,10 @@
 import socket
-
+import sys
 import fido2._pyu2f
 import fido2._pyu2f.base
 
 
+
```

Comment:
This would be flagged if you ran `make check`, which runs black/isort lints. You can auto-fix with `make fix`.

Reply:
I 'll amend the patch and run `make fix`. I 'll also rebase on this branch on top of the latest change to master.